website/content/docs/upgrading/deduplication/acl-policy-templates.mdx

```diff
@@ -0,0 +1,270 @@
+---
+layout: docs
+page_title: Fix ACL policy templates
+description: >-
+  Fix templated ACL behavior for deduplicated user entities and groups.
+---
+
+# Fix ACL policy templates
+
+Fix templated ACL behavior for deduplicated user entities and groups.
+
+**You must address templated ACL policies before enabling forced identity
+deduplication otherwise you may lose access to resources gated by the affected
+policy templates**.
```

Comment:
There is a bunch of nuace here. In many cases I suspect it's pragmatic to go ahead anyway and deal with any fixes needed after the fact. We have to walk the line of making operators aware of the risks, however small, and what they might do if they do hit them, but also don't want to make this process seem totally unfeasable and like way more work than it probably should be - we automated almost all of it with this feature and just need to make sure these edge cases are understood.

With that in mind, I wonder if there is a clear way to message that? I tried to do that originally by giving options for remediation where the first one was "accept the risk and do it anyway" to make it clear that that might be OK for many.

Reply:
I added a new section called "find template policies" that talks about consulting the relevant teams. Let me know if that addresses your concern